deploy.sh

```diff
@@ -86,6 +86,16 @@ function eth_sendTransaction() {
   jsonrpc eth_sendTransaction "$(jq -n $args '[. + $ARGS.named'])" | jq -r
 }
 
+function eth_call(){
```

Comment:
Missing space between `)` and `{`.

Reply:
you think it breaks because of missing space?